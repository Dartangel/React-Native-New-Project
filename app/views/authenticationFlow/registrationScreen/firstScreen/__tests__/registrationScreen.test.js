import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import FirstScreen from '../firstScreen';

describe('FirstScreen', () => {
    test('FirstScreen snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<FirstScreen />);
        expect(snapshot).toMatchSnapshot();
    });
});