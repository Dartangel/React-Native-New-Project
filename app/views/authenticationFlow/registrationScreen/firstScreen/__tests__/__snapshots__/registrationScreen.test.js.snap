// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FirstScreen FirstScreen snapshot 1`] = `
<ImageBackground
  source={
    Object {
      "testUri": "../../../app/src/img/splashScreen1.jpeg",
    }
  }
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <FirstButtons
      onPress={[Function]}
      title="Enter"
    />
    <FirstButtons
      onPress={[Function]}
      title="Registration"
    />
  </View>
</ImageBackground>
`;
